modules/ROOT/pages/config-alerts.adoc

```diff
@@ -6,92 +6,151 @@ endif::[]
+Default alerts are listed in the following table:
+
 [%header%autowidth.spread]
 .Alert Definitions
 |===
-| Name | Description | Threshold | Config map
-| High CPU | Measures the total CPU usage on all cluster nodes | Warning 75%, Critical 90% | kapacitor-alerts 
-| High Memory | Measures the memory consumption on all cluster nodes | Warning 75%, Critical 90% | kapacitor-alerts 
-| Filesystem Usage	| Measures the file system usage | Warning 80%, Critical 90% | kapacitor-alerts 
-| Kubernetes Node Status | Checks Kubernetes status for all nodes| When Kubernetes reports a `NotReady` node status | kapacitor-alerts 
-| Etcd Latency | Measures the latency between nodes, which can affect `etcd` performance | Warning 521 seconds, Critical 1024 seconds | kapacitor-alerts 
-| Systemd health | Checks if the `systemd` slice (where gravity is running) is healthy | When gravity or child systems, such as `docker`, `etcd`, `kube-proxy`, `kube-kubelet`, `planet-agent`, `dnsmasq`, and `flanneld`, are down | kapacitor-alerts 
-| Influxdb Healthcheck Status | Checks the health of `Influxdb` (where the metrics for monitoring are stored) | No connection to `Influxdb` | kapacitor-alerts 
-| Networking Configuration | Checks networking operating system flag status | When `ip_forward` or `br_netfilter` flags are disabled | kapacitor-alerts 
-| Node Uptime | Checks if nodes are online | When a node is down for more than five minutes | kapacitor-alerts 
-| Etcd Health | Check the status of `etcd` | When `etcd` is down or not healthy | kapacitor-alerts 
-| Docker Status	| Verifies that the docker service is running on each node | When docker service is not running on a node | kapacitor-alerts 
-| Backup Failing | Checks the result of a backup operation | When a backup operation fails | backup-status-alert
-| Stolon Replication Lag | Verifies that `Postgres` replicas are in sync | When a replica is lagging by 1800 or more seconds | kapacitor-alerts 
-| Cassandra Load | Verifies that Cassandra nodes are in sync | Warning 10% difference, Critical 20% difference | cassandra-load
+| Component | Alert | Description
+| CPU | High CPU usage | Triggers a warning when > 75% used, triggers a critical error when > 90% used
+| Memory | High memory usage | Triggers a warning when > 80% used, triggers a critical error when > 90% used
+| Systemd | Overall systemd health | Triggers an error when systemd detects a failed service
+| Systemd  | Individual systemd unit health | Triggers an error when a systemd unit is not loaded or active
+| Filesystem | High disk space usage | Triggers a warning when > 80% used, triggers a critical error when > 90% used
+| Filesystem | High inode usage | Triggers a warning when > 90% used, triggers a critical error when > 95% used
+| System | Uptime | Triggers a warning when a node's uptime is less than 5 minutes
+| System | Kernel parameters | Triggers an error if a parameter is not set. See [value matrix](requirements.md#kernel-module-matrix) for details.
```

Comment:
@alejandrorascovan this isn't a valid link for our docs. Where is this coming from?

Reply:
I copied and pasted it from Gravitational's website. Are you saying this is not a valid table? Do you mind fixing it? It looked fine on the preview of my editor